etc/telegraf.conf

```diff
@@ -430,8 +430,42 @@
 #   # listen = ":9126"
 
 
-# # Configuration for the Riemann server to send metrics to
+# # Configuration for Riemann server to send metrics to
 # [[outputs.riemann]]
+#   ## Address of the Riemann server
+#   address = "localhost:5555"
+#
+#   ## Transport protocol to use, either tcp or udp
+#   transport = "tcp"
+#
+#   ## Riemann TTL, floating-point time in seconds.
+#   ## Defines how long that an event is considered valid for in Riemann
+#   # ttl = 30.0
+#
+#   ## Separator to use between measurement and field name in Riemann service name
+#   separator = "/"
+#
+#   ## Set measurement name as a Riemann attribute,
+#   ## instead of prepending it to the Riemann service name
+#   # measurement_as_attribute = false
```

Comment:
Can you provide more details on what this means? can you provide an example? How does using this option affect the `separator` option?